tests/gnrc_netif/main.c

```diff
@@ -75,626 +75,619 @@ static void _set_up(void)
 {
     msg_t msg;
 
-    if (ethernet_netif != NULL) {
-        memset(ethernet_netif->ipv6.addrs_flags, 0,
-               sizeof(ethernet_netif->ipv6.addrs_flags));
-        memset(ethernet_netif->ipv6.addrs, 0,
-               sizeof(ethernet_netif->ipv6.addrs));
-        memset(ethernet_netif->ipv6.groups, 0,
-               sizeof(ethernet_netif->ipv6.groups));
-    }
-    if (ieee802154_netif != NULL) {
-        memset(ieee802154_netif->ipv6.addrs_flags, 0,
-               sizeof(ieee802154_netif->ipv6.addrs_flags));
-        memset(ieee802154_netif->ipv6.addrs, 0,
-               sizeof(ieee802154_netif->ipv6.addrs));
-        memset(ieee802154_netif->ipv6.groups, 0,
-               sizeof(ieee802154_netif->ipv6.groups));
-    }
+    memset(ethernet_netif.ipv6.addrs_flags, 0,
+           sizeof(ethernet_netif.ipv6.addrs_flags));
+    memset(ethernet_netif.ipv6.addrs, 0,
+           sizeof(ethernet_netif.ipv6.addrs));
+    memset(ethernet_netif.ipv6.groups, 0,
+           sizeof(ethernet_netif.ipv6.groups));
+
+    memset(ieee802154_netif.ipv6.addrs_flags, 0,
+           sizeof(ieee802154_netif.ipv6.addrs_flags));
+    memset(ieee802154_netif.ipv6.addrs, 0,
+           sizeof(ieee802154_netif.ipv6.addrs));
+    memset(ieee802154_netif.ipv6.groups, 0,
+           sizeof(ieee802154_netif.ipv6.groups));
+
     for (unsigned i = 0; i < DEFAULT_DEVS_NUMOF; i++) {
-        if (netifs[i] != NULL) {
-            memset(netifs[i]->ipv6.addrs_flags, 0,
-                   sizeof(netifs[i]->ipv6.addrs_flags));
-            memset(netifs[i]->ipv6.addrs, 0, sizeof(netifs[i]->ipv6.addrs));
-            memset(netifs[i]->ipv6.groups, 0, sizeof(netifs[i]->ipv6.groups));
-        }
+        memset(netifs[i].ipv6.addrs_flags, 0,
+               sizeof(netifs[i].ipv6.addrs_flags));
+        memset(netifs[i].ipv6.addrs, 0, sizeof(netifs[i].ipv6.addrs));
+        memset(netifs[i].ipv6.groups, 0, sizeof(netifs[i].ipv6.groups));
     }
+
     /* empty message queue */
     while (msg_try_receive(&msg) > 0) {}
 }
 
 static inline void _test_init(gnrc_netif_t *netif)
 {
-    (void)netif;
-    gnrc_netif_default_init(netif);
-    init_called = true;
+(void)netif;
```

Comment:
Bitte fügen Sie die Einzüge hier und unten wieder hinzu. Dies wird die Anzahl der Änderungen in dieser Datei stark reduzieren.

Reply:
ok, kein Problem!

Reply:
Fixed. It seems I pressed some wrong keys while editing on vim

Reply:
Ups, sorry for the comment in wrong language. I was thinking faster than writing too fast :wink:

Reply:
no hay problema :)